pkg/app/app.go

```diff
@@ -0,0 +1,30 @@
+package app
+
+import "github.com/giantswarm/cluster-operator/service/controller/key"
+
+var (
+	Default = key.AppSpec{
+		Catalog:         "default",
+		Namespace:       "kube-system",
+		UseUpgradeForce: true,
+	}
+
+	Exceptions = map[string]key.AppSpec{
+		"cert-exporter": {
+			Chart: "cert-exporter",
+		},
+		"chart-operator": {
+			// chart-operator must be installed first so the chart CRD is
+			// created in the tenant cluster.
```

Comment:
I think the comment should be for line 16. But will this work? The for loop in the apps resource doesn't seem to be sorted.

Reply:
This is map array, so it's okay even apps in loop statement is not sorted.

Reply:
OK got it. Let's not worry about sorting for now.

But please check how long it takes to install apps in new tenant clusters. If these changes slow that down a lot we might need to do something.